n3fit/src/n3fit/backends/keras_backend/internal_state.py

```diff
@@ -64,8 +82,7 @@ def set_number_of_cores(max_cores=None, max_threads=None):
 
     log.info("Setting the number of cores to: %d", cores)
     try:
-        tf.config.threading.set_inter_op_parallelism_threads(threads)
-        tf.config.threading.set_intra_op_parallelism_threads(cores)
+        set_threading(threads, cores)
     except RuntimeError:
         # If pdfflow is being used, tensorflow will already be initialized by pdfflow
         # maybe it would be good to drop completely pdfflow before starting the fit? (TODO ?)
```

Comment:
What causes the runtimeerror when I don't have tensorflow but use torch instead? The log.warning below seems incorrect

Reply:
There should be no runtime error in that case (partly because with torch I'm not setting the threads, but also because there will no conflict).

The warning is written really badly but it is not incorrect I think?